code/src/components/Form.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+import Question from "components/Question";
+import Button from "components/Button";
+import SubmittedView from "components/SubmittedView";
+import "./Form.css";
+
+const questions = [
+  {
+    id: "preference",
```

Comment:
`"preference"` feels more like a `name` rather than an `id`. We can mock them with libs like [these](https://www.npmjs.com/package/uuid-random). They are never exposed or rendered to the user, but used for managing items during runtime e.g. 
``` jsx
[ { id: UUID(), value: 1 }, { id: UUID(), value: 2 } ].map(item => <div key{item.id}>{item.value}</div>)
```

Reply:
Oh, awesome thanks for the library suggestion! I'll definitely give it a go and generate IDs next time.